src/lib/rehype/rehypeImgSize.ts

```diff
@@ -0,0 +1,69 @@
+import path from "path"
+import { visit } from "unist-util-visit"
+import sizeOf from "image-size"
+import type { Plugin } from "unified"
```

Comment:
where do `Plugin` and `Root` come from? I don't see `unified` or `hast` dependencies in the package.json

Reply:
Yea...those are deps are used and installed by `next-mdx-remote`. Since these were just types I didn't include them but I think we could add them as dev deps. Will add them.